types/command-line-usage/command-line-usage-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as commandLineUsage from "command-line-usage";
```

Comment:
You should import modules with `export =` using the `import commandLineUsage = require("command-line-usage"`